sotodlib/site_pipeline/update_obsdb.py

```diff
@@ -218,17 +252,18 @@ def main(config:str,
                     logger.error(f"No {coor} pointing in some streams for obs_id {obs_id}")
                     pass
             try:
-                if index.get("telescope_flavor")=="SAT":
+                if very_clean.get("telescope_flavor")=="sat":
```

Comment:
With the change above, you can just do `if very_clean['telescope_flavor'] == 'sat':`.  Note again that == is an operator and gets spaces around it.